### PR TITLE
Add new features to demo

### PR DESCRIPTION
**Pull request by karandatwani92**
Solves https://github.com/Laravel-Backpack/demo/issues/526

## Checklist(Feel free to edit )
- [x] Themes
- [x] Dark mode
- [x] Layouts
- [x] Dropzone` Field
- [x] [Emun Field](https://backpackforlaravel.com/docs/6.x/crud-fields#enum-1)
- [ ] [checklist_dependency Field](https://backpackforlaravel.com/docs/6.x/crud-fields#checklist_dependency)[❌Why Not](https://github.com/Laravel-Backpack/CRUD/issues/5143#issuecomment-1607909727)
- [ ] [checklist_dependency Column](https://backpackforlaravel.com/docs/6.x/crud-columns#checklist_dependency-1) [❌Why Not](https://github.com/Laravel-Backpack/CRUD/issues/5143#issuecomment-1607909727)
- [x] 23 New Columns
- [x] `closure` column
- [x] `enum` column
- [x] `view` column
- [ ] [row_number column](https://backpackforlaravel.com/docs/6.x/crud-columns#row_number-1) [❌Fix Required](https://github.com/Laravel-Backpack/CRUD/issues/5215)
- [x] `switch` Column
- [x] [browse_multiple Column](https://backpackforlaravel.com/docs/6.x/crud-columns#browse_multiple-pro)
- [x] `dropzone` Column
- [x] `repeatable` Column
- [x] `video` Column
- [x] `date_range` Column
- [x] [Uploaders/withFiles()](https://backpackforlaravel.com/docs/6.x/crud-uploaders)
- [ ] [Medialibrary-uploaders](https://github.com/laravel-backpack/medialibrary-uploaders)❌Uploads Disabled
- [x] Custom form operation
- [x] Quick buttons
- [x] [Show Operation Tabs](https://backpackforlaravel.com/docs/6.x/crud-operation-show#tabs-display-columns-in-tabs)
- [x] Check `icon-picker`
- [ ] [CSS hooks](https://backpackforlaravel.com/docs/6.x/release-notes#css-hook-classes)
- [x] Remove: `address_algolia`

### CODE REVIEW

**Comment by karandatwani92** [comment]
Hey @tabacitu, please review. I hope you find it well.

And please put some light on [CSS hooks](https://backpackforlaravel.com/docs/6.x/release-notes#css-hook-classes). Docs are missing too.

**Comment by tabacitu** [comment]
@pxpm please review and merge if everything's ok. If needed, feel free to make changes to speed up the merge.

We can have separate PRs for whatever's left.

**Comment by pxpm** [comment]
I cannot test this while this is not final. Please re-assign me when there are no more changes @karandatwani92 

Thanks 🙏

**Comment by karandatwani92** [comment]
@pxpm no more changes, please review

File: app/Http/Controllers/Admin/MonsterCrudController.php [hunk 1]
```diff
@@ -91,27 +105,32 @@ public function setupListOperation()
                 'function' => function ($entry) {
                     return 'Created on '.$entry->created_at;
                 },
+                'tab' => 'Function/Closure',
```
Comment (pxpm) [inline app/Http/Controllers/Admin/MonsterCrudController.php:108]: I think we can move this to miscellaneous or something similar, no need for a tab for this function/closure thing.

File: app/Http/Controllers/Admin/MonsterCrudController.php [hunk 2]
```diff
@@ -125,17 +144,20 @@ public function setupListOperation()
                     $query->orWhere('text', 'like', '%'.$searchTerm.'%');
                 },
                 'escaped'       => true,
+                'tab'           => 'Function/Closure',
```
Comment (pxpm) [inline app/Http/Controllers/Admin/MonsterCrudController.php:147]: I think we can move this to miscellaneous or something similar, no need for a tab for this function/closure thing.

File: app/Http/Controllers/Admin/MonsterCrudController.php [hunk 3]
```diff
@@ -174,29 +199,40 @@ public function setupListOperation()
...
+            'label'   => 'Quick Button',
+            'icon'    => 'la la-fast-forward',
+            'wrapper' => [
+                // 'element' => 'a',
+                'href'   => url('something'),
+                'target' => '_blank',
+                'title'  => 'Create a new email to this user',
+            ],
+        ]);
+
```
Comment (pxpm) [inline app/Http/Controllers/Admin/MonsterCrudController.php:229]: let's add a button that does something, I don't know: `Go to Backpack Website` or `Go to Documentation` or something similar. But that when clicked something "real" happen.

File: app/Http/Controllers/Admin/MonsterCrudController.php [hunk 4]
```diff
@@ -205,12 +241,71 @@ public function setupShowOperation()
...
+        $this->crud->addColumn([
+            'name'    => 'wysiwyg',
+            'type'    => 'wysiwyg',
+            'label'   => 'Wysiwyg',
+            'tab'     => 'WYSIWYG Editors',
+        ]);
+
+        $this->crud->addColumn([
```
Comment (pxpm) [inline app/Http/Controllers/Admin/MonsterCrudController.php:275]: Do we have this features field ?

File: app/Http/Controllers/Admin/MonsterCrudController.php [hunk 5]
```diff
@@ -240,76 +337,97 @@ public function setupShowOperation()
             'type'          => 'model_function_attribute',
             'function_name' => 'getCategory', // the method in your Model
             'attribute'     => 'name',
+            'tab'           => 'Function/Closure',
```
Comment (pxpm) [inline app/Http/Controllers/Admin/MonsterCrudController.php:340]: I think we can move this to miscellaneous or something similar, no need for a tab for this function/closure thing.

File: app/Http/Controllers/Admin/MonsterCrudController.php [hunk 6]
```diff
@@ -240,76 +337,97 @@ public function setupShowOperation()
...
             'name'      => 'id',
             'type'      => 'number',
             'label'     => '#',
             'orderable' => false,
+            'tab'       => 'Simple',
         ]);
 
         $this->crud->addColumn([
             'name'  => 'browse_multiple',
             'type'  => 'array',
```
Comment (pxpm) [inline app/Http/Controllers/Admin/MonsterCrudController.php:407]: we have the browse_multiple column right ? Why not use it ?

File: database/migrations/2023_07_16_114745_rename_address_to_google_column_from_monsters.php [hunk 7]
```diff
@@ -0,0 +1,22 @@
+<?php
```
Comment (pxpm) [inline database/migrations/2023_07_16_114745_rename_address_to_google_column_from_monsters.php:1]: Not sure we really need to rename anything here, it's just adding complexity to the PR. 

If we do it, we do it in a separate PR. renaming the column has no benefit in this PR

File: app/Http/Controllers/Admin/MonsterCrudController.php (hunk 3 above)
Comment (karandatwani92) [reply app/Http/Controllers/Admin/MonsterCrudController.php:229]: OK Right, I'm targeting it to https://backpackforlaravel.com/docs/crud-buttons#creating-a-quick-button-1
That matches the button title and makes the point.
let me know what you think.

File: app/Http/Controllers/Admin/MonsterCrudController.php (hunk 4 above)
Comment (karandatwani92) [reply app/Http/Controllers/Admin/MonsterCrudController.php:275]: Yes we have a `features` field of the `repeatable` type

File: database/migrations/2023_07_16_114745_rename_address_to_google_column_from_monsters.php (hunk 7 above)
Comment (karandatwani92) [reply database/migrations/2023_07_16_114745_rename_address_to_google_column_from_monsters.php:1]: Removing it!

File: app/Http/Controllers/Admin/MonsterCrudController.php (hunk 5 above)
Comment (karandatwani92) [reply app/Http/Controllers/Admin/MonsterCrudController.php:340]: Done

File: app/Http/Controllers/Admin/MonsterCrudController.php (hunk 2 above)
Comment (karandatwani92) [reply app/Http/Controllers/Admin/MonsterCrudController.php:147]: Done

File: app/Http/Controllers/Admin/MonsterCrudController.php (hunk 6 above)
Comment (karandatwani92) [reply app/Http/Controllers/Admin/MonsterCrudController.php:407]: Changed to `browse_multiple

**Comment by karandatwani92** [comment]
@pxpm Requested changes added.

File: resources/views/vendor/backpack/crud/columns/something.blade.php [hunk 8]
```diff
@@ -0,0 +1,2 @@
+{{-- Used for 'custom_view' column example --}}
+<s>Something</s>
```
Comment (pxpm) [inline resources/views/vendor/backpack/crud/columns/something.blade.php:2]: Let's call this `custom_column_example`. the comment on the file helps. Nice touch 🙏 
 And maybe add an icon here too along with some text (maybe **half-bold**~~half-razored~~ ) to show that is a different flexible view.

Comment (karandatwani92) [reply resources/views/vendor/backpack/crud/columns/something.blade.php]: Done

File: app/Http/Controllers/Admin/Operations/SMSOperation.php [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+    {
+        $this->formDefaults(
+            operationName: 'SMS',
+            buttonStack: 'line', // alternatives: top, bottom
+            buttonMeta: [
+                'icon'    => 'las la-sms',
+                'label'   => 'SMS',
+                'wrapper' => [
+                    'target' => '_blank',
+                ],
```
Comment (pxpm) [inline app/Http/Controllers/Admin/Operations/SMSOperation.php:42]:
```suggestion
```
